Filter/AbstractDateFilter.php

```diff
@@ -47,10 +47,6 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
                 return;
             }
 
-            if (!$data['value']['start'] || !$data['value']['end']) {
-                return;
-            }
```

Comment:
I don't really think you should remove this block, I think you should just replace `||` with `&&` here.